modin/pandas/test/utils.py

```diff
@@ -46,10 +46,6 @@
 from modin.config import MinPartitionSize, NPartitions, TestDatasetSize, TrackFileLeaks
 from modin.utils import to_pandas, try_cast_to_pandas
 
-# Flag activated on command line with "--extra-test-parameters" option.
-# Used in some tests to perform additional parameter combinations.
-extra_test_parameters = False
```

Comment:
The new approach is more native for pytest and does not require the creation of global variables (like the old one).